forte/train_preprocessor.py

```diff
@@ -0,0 +1,355 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
I forgot to mention we previously often miss the module docstring. Basically a docstring specifying what this module contains.

Reply:
Is there any example for module docstring?

Reply:
Yes like that. I realize it is a PEP8 requirement only recently.

Reply:
Well, I have added them anyway.

Reply:
Here is another example:
https://github.com/asyml/forte/blob/master/forte/processors/eliza_processor.py#L14

I think it let people skim through the doc without scrolling a lot.